src/components/drawer/ComponentDetailPanel.vue

```diff
@@ -154,3 +209,9 @@ onUnmounted(() => {
   pluginEditSubscription.unsubscribe();
 });
 </script>
+
+<style lang="scss">
+.q-card__section--vert {
+  padding: 0px;
```

Comment:
are you sure to know what you're doing ?

Reply:
changed